sympy/algebras/quaternion.py

```diff
@@ -720,3 +720,422 @@ def to_rotation_matrix(self, v=None):
+        >>> q.angle()
+        pi/2
+
+        """
+
+        return atan2(self.vector_part().norm(), self.scalar_part())
+
+    @classmethod
+    def coplanar(cls, q1, q2):
+        """
+        Returns True if the plane of the two quaternions are coplanar else returns False.
+
+        Explanation
+        ===========
+
+        checks if the 3D vectors written as quaternions are in the same plane.
+        In order to return a True, the quaternions and the origin should be in the
+        same plane, and the real part must be zero for it to make sense.
+
+        Parameters
+        ==========
+
+        q1 : a Quaternion
+        q2 : a Quaternion
+
+        Examples
+        ========
+
+        >>> from sympy.algebras.quaternion import Quaternion
+        >>> q1 = Quaternion(1, 4, 4, 4)
+        >>> q2 = Quaternion(2, 8, 8, 8)
+        >>> Quaternion.coplanar(q1, q2)
+        True
+
+        >>> q1 = Quaternion(2, 8, 13, 12)
+        >>> Quaternion.coplanar(q1, q2)
+        False
+
+        See Also
+        ========
+
+        ternary_coplanar
+        is_pure
+
+        """
+        if (q1.is_zero_quaternion()) or (q2.is_zero_quaternion()):
+            raise ValueError('Any of the provided quaternions must not be 0')
+
+        aq = q1.axis()
+        ar = q2.axis()
+
+        return fuzzy_or([(aq - ar).is_zero_quaternion(), (aq + ar).is_zero_quaternion()])
+
+    @classmethod
+    def ternary_coplanar(cls, q1, q2, q3):
+        """
+        Returns True if the axis of the pure quaternions seen as 3D vectors
+        q1, q2, and q3 are coplanar else returns False.
+
+        Explanation
+        ===========
+
+        Two general quaternions are coplanar (in this arc sense) when their axes are parallel.
+
+        Parameters
+        ==========
+
+        q1 : a pure Quaternion.
+        q2 : a pure Quaternion.
+        q3 : a pure Quaternion.
+
+        Examples
+        ========
+
+        >>> from sympy.algebras.quaternion import Quaternion
+        >>> q1 = Quaternion(0, 4, 4, 4)
+        >>> q2 = Quaternion(0, 8, 8, 8)
+        >>> q3 = Quaternion(0, 24, 24, 24)
+        >>> Quaternion.ternary_coplanar(q1, q2, q3)
+        True
+
+        >>> q1 = Quaternion(0, 8, 16, 8)
+        >>> q2 = Quaternion(0, 8, 3, 12)
+        >>> Quaternion.ternary_coplanar(q1, q2, q3)
+        False
+
+        See Also
+        ========
+
+        axis
+        is_pure
+
+        """
+
+        if fuzzy_not(q1.is_pure()) or fuzzy_not(q2.is_pure()) or fuzzy_not(q3.is_pure()):
+            raise ValueError('The provided quaternions must be pure')
+
+        M = Matrix([[q1.b, q1.c, q1.d], [q2.b, q2.c, q2.d], [q3.b, q3.c, q3.d]]).det()
+        return M.is_zero
+
+    def parallel(self, other):
+        """
+        Returns True if the two pure quaternions seen as 3D vectors are parallel else returns False.
+
+        Explanation
+        ===========
+
+        Two pure quaternions are called parallel when their product is commutative.
```

Comment:
Is there a more intuitive way of explaining what this means?

From the example below it seems that they are parallel if one is a scalar (real) multiple of the other.